Modules/_zstd/_zstdmodule.c

```diff
@@ -554,28 +554,63 @@ static PyMethodDef _zstd_methods[] = {
     {0}
 };
 
-
-static inline int
-add_vars_to_module(PyObject *m)
+static int
+_zstd_exec(PyObject *m)
```

Comment:
One way to make it easier to maintain maybe is to have more than one function, each of which responsible for adding specific components. For instance, one function for adding exceptions, one for adding types, one for adding constants, etc, instead of a huge `_exec` which sometimes gets tricky.  This is only a suggestion and it's really up to who maintains this module (not me)

Reply:
I think this is better than the previous, where `_exec()` was setting up everything but the module constants, but we could indeed split to several smaller functions. The other compression modules seem to use a single exec function, though, so perhaps best to keep as is for now? Happy to revisit later though.